openedx/features/course_experience/utils.py

```diff
@@ -254,6 +254,21 @@ def get_resume_block(block):
     return block
 
 
+def get_start_block(block):
+    """
+    Gets the deepest block to use as the starting block.
+    """
+    if not block.get('children'):
+        return block
+
+    first_child = block['children'][0]
+
+    if not first_child.get('children'):
+        return first_child
```

Comment:
Isn't this redundant with the check at the top of the func?

Reply:
silly me